src/serac/physics/utilities/functional/domain_integral_shared.hpp

```diff
@@ -0,0 +1,163 @@
+#pragma once
+
+#include "serac/physics/utilities/functional/tensor.hpp"
+
+namespace serac {
+
+namespace domain_integral {
+
+/**
+ * @brief Computes the arguments to be passed into the q-function (shape function evaluations)
+ * By default:
+ *  H1 family elements will compute {value, gradient}
+ *  Hcurl family elements will compute {value, curl}
+ *  TODO: Hdiv family elements will compute {value, divergence}
+ *  TODO: L2 family elements will compute value
```

Comment:
```suggestion
 *  L2 family elements will compute value
```

Reply:
Were the resolved comments actually resolved? I don't see the changes.